### ISSUE TITLE
feat: Move Contributing-guidelines and  Code-of-conduct to root directory

### ISSUE THREAD

**Issue by tobySolutions** [aidenybai/million#325]
**Is your feature request related to a problem? Please describe.**
I believe to follow convention and for easy accessibility, the code of conduct and the contributing guidelines should be in the root directory of the project

**Describe the solution you'd like**
I moved both files to the root directory and some work will still need to be done on the `README.md` to fix the linking to both files

**Additional context**
Check out [Solid](https://github.com/solidjs/solid), [Linkfree](https://github.com/EddieHubCommunity/LinkFree)

## What we have now: 

![Screenshot from 2023-04-26 14-12-17](https://user-images.githubusercontent.com/96334363/234587334-912b6fec-29b5-48fb-83d9-18af43b8c623.png)

## What it should be like:
![Screenshot from 2023-04-26 14-13-06](https://user-images.githubusercontent.com/96334363/234587615-c7e728a8-1a3e-4aa5-ae03-e4831a1f6b54.png)

**Comment by aidenybai** [comment]
I actually disagree with this change--i think all contribution related resources should be under `.github/`

**Comment by tobySolutions** [comment]
> I actually disagree with this change--i think all contribution related resources should be under `.github/` to avoid clutter at the root level @tobySolutions

Ok sure, no problem. Reversing the changes now.
